src/components/data-table/sort_filter_worker.ts

```diff
@@ -1,57 +1,34 @@
-import memoizeOne from "memoize-one";
-// eslint-disable-next-line import/no-cycle
-import {
-  DataTableColumnContainer,
-  DataTableColumnData,
+// To use comlink under ES5
+import "proxy-polyfill";
+import { expose } from "comlink";
+import type {
+  DataTableSortColumnData,
   DataTableRowData,
   SortingDirection,
+  HaDataTable,
 } from "./ha-data-table";
 
-export const filterSortData = memoizeOne(
-  async (
-    data: DataTableRowData[],
-    columns: DataTableColumnContainer,
```

Comment:
This type was wrong, I replaced that with type `SortableColumnContainer` which I extracted from `HaDataTable["_sortColumns"]`